kayenta-web/src/test/java/com/netflix/kayenta/controllers/BaseControllerTest.java

```diff
@@ -74,21 +83,41 @@ StorageServiceRepository storageServiceRepository(List<StorageService> storageSe
     }
 
     @Bean
-    StorageService storageService() {
-      StorageService mock = mock(StorageService.class);
```

Comment:
Because of how this was wired up it was being shared across all tests. Swapped it to be MockBean which resets after every test